src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url('../images/star.svg') no-repeat;
+    background-position: center;
+    width: $starsSize;
+    height: $starsSize;
+    margin-right: $marginStars;
```

Comment:
Ensure that the variables '$starsSize' and '$marginStars' are defined in your SCSS files. If they are not defined, the SCSS compilation will fail.